server/server.js

```diff
@@ -36,12 +39,30 @@ if (process.env.NODE_ENV === "production") {
 
 app.use(
   "/graphql",
-  cors({ origin: "*" }),
```

Comment:
This allows us to send API requests from front end once hosted at Heroku

Reply:
actually your frontend will be build using heroku-postinstall in package json in the root
you should validate that heroku is building the client since both client and server will be deployed from one single heroku app with one single url

Reply:
if it does not work, then change heroku-postinstall to heroku-postbuild